pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/FunctionCommon.java

```diff
@@ -383,4 +377,112 @@ public static double roundDecimal(double value, int places) {
         double scale = Math.pow(10, places);
         return Math.round(value * scale) / scale;
     }
+
+    public static ClassLoader getClassLoaderFromPackage(
+            org.apache.pulsar.functions.proto.Function.FunctionDetails.ComponentType componentType,
+            String className,
+            File packageFile,
+            String narExtractionDirectory) {
+        String connectorClassName = className;
+        ClassLoader classLoader;
```

Comment:
instead of having this variable and return it at the end. I suggest return the classLoader at the end of each if/else clause. Easier to read

Reply:
sure